### DISCUSSION TITLE
"message":"401

### DISCUSSION THREAD

**Discussion by HungryVovka** [victornpb/undiscord#478] [4. Bugs and Issues]
Used version 5.2.0 without any problems. Google Chrome Browser, Tampermonkey Browser Extension.

Installed version 5.2.1 an hour ago. Every time I try to use this, this error appears on the screen: 
Error searching messages, API responded with status 401! {"message":"401: Unauthorized","code":0} CoreException {}

Rolled back to version 5.2.0, everything works fine again, as before.

**Comment by victornpb** [comment]
does the error appear right away?

**Reply by Ayoub-JD** [reply]
for me, it does show right away "API responded with status 401!"

**Reply by Deleterman21** [reply]
It appears instantly.

**Reply by michaelhabieb** [reply]
i tried it on my pc and laptop could it have to do with it not working if the other peson blocked you?

**Reply by LightZirconite** [reply]
how to do it

**Comment by Deleterman21** [comment]
Same here.

**Comment by arjunkumar23** [comment]
where can I get the 5.2.0 version, unable to find it

**Reply by FoodyFood** [reply]
https://greasyfork.org/en/scripts/406540-undiscord?version=1159585

**Comment by No-1l** [comment]
Same i m having this :/ 
![image](https://user-images.githubusercontent.com/129275441/228497186-898a4e21-1c16-4ab8-827e-1c0142253904.png)

**Reply by OnFriday1224** [reply]
This is what I'm getting, just started a couple days ago

![Discord Error](https://user-images.githubusercontent.com/120187640/228637957-c2d43fda-ea31-4509-a663-6fb9344d979e.png)

**Comment by tuxicorn** [comment]
same

**Comment by zaberwookie** [comment]


**Reply by victornpb** [reply]
LOL.

Discord made a change on their end that broke undiscord, I’m having to work around that issue, a fix is in the works 

So no, I’m not Adobe or Activision or EA 😂

it's not going to be paid.

**Reply by coltschultz** [reply]
thanks for your hard work!

**Reply by paultsanghy** [reply]
any updates on the 25 messages limit?

**Reply by soosannn** [reply]
Just hard refresh the page and do it again \m/

**Reply by BradAlright** [reply]
GDAMMIT I can't believe how stupid this was. Lol. Thanks for saving us, bud.

**Reply by ItsSunee** [reply]
Hope it gets fixed soon!

**Comment by GitExploraa** [comment]
same here, any updates please?

**Comment by otchix** [comment]
I fixed it by going into advanced settings in Undiscord and click fill next to Authorization Token, somehow it was filled with the wrong Token..

**Reply by TheQueenofKings** [reply]
I tried this but it gave me a different error afterwards
```Fetching messages...
Error searching messages, API responded with status 400!
	{"code":50035,"errors":{"max_id":{"_errors":[{"code":"NUMBER_TYPE_COERCE","message":"Value \"email-address\" is not snowflake."}]}},"message":"Invalid Form Body"}
CoreException	{}

**Reply by otchix** [reply]
In my case chrome somehow autofilled Autherization token and messages interval with my discord password i guess (Don't know what else it could be but it was filled by password manager in chrome). I only had this issue on latest version of Undiscord. Maybe check if all settings are filled correctly.

**Reply by p0pdr0p** [reply]
Yeah this happened to me as well, as soon as I deleted the autofill information it worked

**Reply by cruzluna** [reply]
Make sure to delete all of the autofill stuff. My issue was the interval of messages auto-filling and the authorization token.

**Reply by cruzluna** [reply]
You need to clear your email from the Messages interval.

**Comment by 1030** [comment]
Worked for me too, I deleted the remembered password in chrome and bingo works great after that.

**Comment by TheQueenofKings** [comment]
Still doesn't work for me. I don't know what to do anymore. :/

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by ltriv131** [comment]
I figured it out! This is a combo of replies. As Otchix replied, you should go to advanced->authorization token and click fill. It auto-filled with my password. If you click delete now, it gives you an error 400. This is because it puts your email into the messages interval->before field. Simply delete the text and you should be good to go. This worked for me and it's working great.

**Reply by Lunatic3k** [reply]
That actually works, thanks

**Reply by littletreepot** [reply]
yes, this worked for me! thank you.

**Reply by Hitomiwari** [reply]
Thank you so much.

**Reply by YvesLangstonMays** [reply]
@victornpb

**Reply by Deleterman21** [reply]
Thank you so much this works. @victornpb

**Reply by TheQueenofKings** [reply]
This workS!

**Reply by mohammedc212** [reply]
BRO UR ACTUALLY A LIFE SAVER THANK YOU

**Reply by Notonyyyy** [reply]
thanks thanks thanks this works!

**Comment by PIKUT0** [comment]
Close.

**Comment by GreatLoveV** [comment]
> Used version 5.2.0 without any problems. Google Chrome Browser, Tampermonkey Browser Extension.
> 
> Installed version 5.2.1 an hour ago. Every time I try to use this, this error appears on the screen: Error searching messages, API responded with status 401! {"message":"401: Unauthorized","code":0} CoreException {}
> 
> Rolled back to version 5.2.0, everything works fine again, as before.

Worked after clearing the message intervals input text, ig an email of mine got auto-filled there by accident. Check if its empty or not, might be the cause.

**Comment by BradAlright** [comment]
This happens because your browser's autofill is putting your email into "message interval" and your email password into "auth token". Clear those fields and click to fill them and you're g2g.

**Comment by luphoria** [comment]
> This happens because your browser's autofill is putting your email into "message interval" and your email password into "auth token". Clear those fields and click to fill them and you're g2g.

This, but not quite for me. Autofilling over the token at the end. Hit "fill" again, fixed it. Seems like an easy PR. I can get it fixed later today probably.
